### PR TITLE
Increase etcd index if delete the route root

### PR DESCRIPTION
**Pull request by zhen-w**
https://github.com/zalando/skipper/issues/474

### CODE REVIEW

File: etcd/etcd.go [hunk 1]
```diff
@@ -421,6 +421,9 @@ func (c *Client) LoadUpdate() ([]*eskip.Route, []string, error) {
 		} else if err != nil {
 			return nil, nil, err
 		} else if response.Node.Dir {
+            if response.Node.ModifiedIndex > c.etcdIndex {
+                c.etcdIndex = response.Node.ModifiedIndex
+            }
```
Comment (szuecs) [inline etcd/etcd.go]: @zhen-w Can you indent the code correctly, please?
This will do it automatically:

    $ gofmt -w etcd/etcd.go

**Comment by szuecs** [comment]
Thanks for your PR!

File: etcd/etcd.go (hunk 1 above)
Comment (zhen-w) [reply etcd/etcd.go]: sure. Done

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
👍

**Comment by aryszka** [comment]
thanks a lot for the help! :heart:

**Pull request merged**